Formula/ntp.rb

```diff
@@ -19,6 +19,8 @@ class Ntp < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "16137579ea8978ef71a1222071cd9e44a42eeeaa8d83ed83bdbf0fd048e6178c"
   end
 
+  # Does not build with `openssl@3`
```

Comment:
Arch build it fine with OpenSSL 3.0.8 though, so this we might be doing something wrong here